### ISSUE TITLE
get_element fails for "RNA product"

### ISSUE THREAD

**Issue by cbizon** [biolink/biolink-model-toolkit#20]
BMT 0.3.0, https://raw.githubusercontent.com/biolink/biolink-model/1.3.9/biolink-model.yaml

There is a  class called "RNA product". It shows up as a descendent of named_thing, but get_element can't find it:

```
name = 'RNA product'
print(name in tk.get_descendants('named thing'))
print(tk.get_element(name))
```

```
True
None
```

**Comment by cbizon** [comment]
Looks like this is true for some limb of the model:
```
['RNA product', 'RNA product isoform', 'noncoding RNA product', 'microRNA']
```

**Comment by deepakunni3** [comment]
Yes, this is because of the inconsistency in the way the string for RNA Product itself is represented in the model.

In the model it is defined as `RNA product`. A CamelCase form of this would be `RNAProduct`. 

But using libraries that do such conversion from sentence case to CamelCase is tricky.

Will add a fix for this in `0.3.1`
